src/customers/controller/customer.controller.ts

```diff
@@ -0,0 +1,122 @@
+import {
+  ApiTags,
+  ApiOperation,
+  ApiResponse,
+  ApiParam,
+  ApiBody,
+} from '@nestjs/swagger';
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  HttpCode,
+  Param,
+  Patch,
+  Post,
+} from '@nestjs/common';
+
+import { CustomerService } from '../service/customer.service';
+import { CustomerMapper } from '../mapper/customer.mapper';
+import { CustomerDto } from '../dto/customer.dto';
+import { Customer } from '../domain/customer.entity';
+import { CreateCustomerDto } from '../dto/create-customer.dto';
+import { UpdateCustomerDto } from '../dto/update-customer.dto';
+
+@ApiTags('customers')
```

Comment:
**suggestion(non-blocking):** to avoid magic strings I will suggest having a config file (e.g.: `customers.config.ts`) which exports the const string for the feature name you are using it here for `customers` (and which later could be used in the main.ts if you have different ApiTag from the endpoint route)